lib/active_merchant/billing/gateways/stripe_payment_intents.rb

```diff
@@ -303,9 +309,19 @@ def add_stored_credentials(post, options = {})
         post[:payment_method_options][:card] ||= {}
         post[:payment_method_options][:card][:mit_exemption] = {}
 
-        # Stripe PI accepts network_transaction_id and ds_transaction_id via mit field under card.
-        post[:payment_method_options][:card][:mit_exemption][:network_transaction_id] = stored_credential[:network_transaction_id] if stored_credential[:network_transaction_id]
+        # Stripe PI accepts network_transaction_id and ds_transaction_id via mit field under card. It can be sent in nested under stored credentials OR as its own field (add_ntid handles when it is sent in on its own)
         post[:payment_method_options][:card][:mit_exemption][:ds_transaction_id] = stored_credential[:ds_transaction_id] if stored_credential[:ds_transaction_id]
+        post[:payment_method_options][:card][:mit_exemption][:network_transaction_id] = stored_credential[:network_transaction_id] if stored_credential[:network_transaction_id]
```

Comment:
Do we need to have a check not to set this if options[:network_transaction_id] exists?

Reply:
I believe the `return unless` portion handles that. If the options include [:stored_credentials] this path is taken and the ntid will be set from the network_transaction_id that is inside stored_credentials, but if they do not, it isn't set. Same with the `add_ntid` method below it using `return unless` network_transaction_id is in options.

Reply:
I guess there could be a chance that someone might send in stored_credentials AND a network_transaction_id in the options level. 🤔

Reply:
I ran the test with ntid being sent in under options AND under stored credentials and it still sent the request with the options[:network_transaction_id]. So it looks like the stored credentials will not overwrite the field if it is sent in separately. Thoughts?

Reply:
Sounds good! You can leave the test in there to show what happens in case anyone else has the same question later on, but that was mostly my interest too re: sending in both in the `options` field.